images/win/scripts/Installers/Install-Vsix.ps1

```diff
@@ -0,0 +1,27 @@
+###################################################################################
+##  File:  Install-Vsix.ps1
+##  Desc:  Install the Visual Studio Extensions from toolset.json
+###################################################################################
+
+Import-Module -Name ImageHelpers -Force

```

Comment:
It is auto-imported so don't need to import it manually.

Reply:
Import-Module has been removed